### ISSUE TITLE
manifest-path changes broke travis-cargo on rust-nightly

### ISSUE THREAD

**Issue by bonifaido** [huonw/travis-cargo#18]
https://github.com/rust-lang/cargo/commit/6097937f0b0c3fcbe3f95d4be22b307559d259ec

This change blows up travis-cargo on my end:
```shell
$ travis-cargo build
the manifest-path must be a path to a Cargo.toml file
```
The fix seems to be easy:
```diff
- manifest = Manifest(os.getcwd())
+ manifest = Manifest(os.getcwd() + "/Cargo.toml")
```

**Comment by bonifaido** [comment]
Thank you!

**Comment by huonw** [comment]
Thanks for filing an issue!

**Comment by Luthaf** [comment]
Just ran into this, using travis-cargo 0.1.6. Here is the log: https://travis-ci.org/Luthaf/cymbalum/jobs/80707484. Are there any changes ?

**Comment by huonw** [comment]
@Luthaf #21 fixed it (published as 0.1.8).

**Comment by Luthaf** [comment]
Yes, it is fixed! Thank you!
